release-repo-scripts/bump_dependency.bash

```diff
@@ -455,8 +455,29 @@ for ((i = 0; i < "${#LIBRARIES[@]}"; i++)); do
     DEP_VER=${VERSIONS[$j]}
     DEP_PREV_VER="$((${DEP_VER}-1))"
 
+    # Rule: *plugin2 -> *plugin3
+    # Replace lines like: "find_package(ignition-cmake2)"
+    #               with: "find_package(ignition-cmake3)"
     find . -type f ! -name 'Changelog.md' ! -name 'Migration.md' -print0 | xargs -0 sed -i "s ${DEP_LIB}${DEP_PREV_VER} ${DEP_LIB}${DEP_VER} g"
 
+    # Rule: IGN_PLUGIN_VER 2 -> IGN_PLUGIN_VER 3
+    # Replace lines like: "set(IGN_PLUGIN_VER 2)"
+    #               with: "set(IGN_PLUGIN_VER 3)"
+    find . -type f ! -name 'Changelog.md' ! -name 'Migration.md' -print0 | xargs -0 sed -i "s@IGN_${DEP_LIB}_VER ${DEP_PREV_VER}@\UIGN_${DEP_LIB}_VER ${DEP_VER}@ig"
+
+    # Replace lines like "find_package(ignition-cmake2 2.0.0)"
+    #               with "find_package(ignition-cmake3)"
+    find . -type f -name 'CMakeLists.txt' -print0 | xargs -0 sed -i "s@\(find_package.*${DEP_LIB}\)${DEP_PREV_VER} \+${DEP_PREV_VER}[^ )]*@\1${DEP_VER}@g"
+
+    # Replace lines like "ign_find_package(ignition-math6 VERSION 6.5.0)"
+    #               with "ign_find_package(ignition-math7)"
+    # Preserves other args and handles edge cases:
+    #               like "ign_find_package(ignition-math6 VERSION 6.5.0 REQUIRED)"
+    #               with "ign_find_package(ignition-math6 REQUIRED)"
+    #               like "ign_find_package(ignition-math6 REQUIRED COMPONENTS VERSION 6.10 eigen3)"
+    #               with "ign_find_package(ignition-math7 REQUIRED COMPONENTS eigen3)"
+    find . -type f -name 'CMakeLists.txt' -print0 | xargs -0 sed -i "s@\(find_package.*${DEP_LIB}\)${DEP_PREV_VER}\(.*\) \+VERSION \+${DEP_PREV_VER}[^ )]*@\1${DEP_VER}\2@g"
```

Comment:
I think lines 468-479 need to be before lines 458-461

Reply:
Oh! You are right. I need to move that down all the way :/

Reply:
https://github.com/ignition-tooling/release-tools/pull/700